### PR TITLE
Tweak documentation for generator troubleshooting

### PR DESCRIPTION
**Pull request by Tobbe**
Some tweaks to the documentation introduced with #4849

I think "issue" too strongly implies there's something wrong with our code, which I don't believe there is.

### CODE REVIEW

File: docs/docs/cli-commands.md [hunk 1]
```diff
@@ -1002,7 +1002,7 @@ Notes:
 **Troubleshooting**
 
 If you see `Error: Unknown type: ...`, don't panic!
-It's a known issue with GraphQL type generation.
+It's a known limitation with GraphQL type generation.
```
Comment (jtoar) [inline docs/docs/cli-commands.md:1005]: Much better (and more accurate) wording!

File: docs/docs/schema-relations.md [hunk 2]
```diff
@@ -156,15 +156,15 @@ type Query {
...
+Remember, the first thing to do when you get an error: _read the error message_.
 The key is `Unknown type: "Shelf"`.
 The type of `Book`'s `shelf` field is `Shelf`.
 But we didn't generate the SDL for `Shelf` yet, so it doesn't exist.
 And naturally, types can't be generated for it.
 
 But fear not.
 This should be an easy fix.
-There's two ways you can go about it.
+There are two ways you can go about it.
```
Comment (jtoar) [inline docs/docs/schema-relations.md:167]: I always get this wrong, thanks!

File: docs/docs/schema-relations.md [hunk 3]
```diff
@@ -100,7 +100,7 @@ Almost identical! But now there's an `id` and the SDL/scaffold generators will w
 ## Troubleshooting Generators
 
 Are you getting errors when generating SDLs or scaffolds for your Prisma models?
-There's a known issue in Redwood's GraphQL type generation that happens when generating SDL for or scaffolding out a Prisma model that has relations before the SDL for the related model exists.
+There's a known limitation in Redwood's GraphQL type generation that happens when generating SDL for, or scaffolding out, a Prisma model that has relations, before the SDL for the related model exists.
```
Comment (jtoar) [inline docs/docs/schema-relations.md:103]: Could we do without the last comma? It reads a little awkward. I'm open to italicizing or bolding the rest of the line if you were going for emphasis?
```suggestion
There's a known limitation in Redwood's GraphQL type generation that happens when generating SDL for, or scaffolding out, a Prisma model that has relations _before the SDL for the related model exists_.
```

**Review by jtoar — APPROVED** [review]
Thanks @Tobbe! Everything looks great, had one minor comment but feel free to ignore

File: docs/docs/schema-relations.md (hunk 3 above)
Comment (Tobbe) [reply docs/docs/schema-relations.md]: Yeah, you're right, it's better without that last comma. Thanks :)

**Pull request merged**